packages/editor/src/plugins/image/editor.tsx

```diff
@@ -84,23 +91,22 @@ export function ImageEditor(props: ImageProps) {
         )}
         data-qa="plugin-image-editor"
       >
-        {hasFocus && showInlineImageUrl ? (
-          <div className="absolute left-side top-side z-[3]">
-            <InlineSrcControls {...props} urlInputRef={urlInputRef} />
-          </div>
-        ) : null}
-
-        <ImageRenderer
-          image={{
-            src,
-            href: state.link.defined ? state.link.href.value : undefined,
-            alt: state.alt.defined ? state.alt.value : undefined,
-            maxWidth: state.maxWidth.defined ? state.maxWidth.value : undefined,
-          }}
-          caption={renderCaption()}
-          placeholder={renderPlaceholder()}
-          forceNewTab
-        />
+        {src.length === 0 && <ImageSelectionScreen {...props} />}
```

Comment:
If the user starts typing in the URL (unlikely, I know) instead of copy/pasting it, the src field will disappear and there will be a broken image icon.

Reply:
Will discuss with @GregorZupan

Reply:
Is there an elegant way to avoid that?

Reply:
>Is there an elegant way to avoid that?

There should be a validation of the entered url. This also applies to copy pasting. 
Validation should at least check if the url is structured like a valid image link.
(starting with "http" and ending with a supported file format like "jpg" for example)
(but maybe also only allow images from selected domains or fetch the image to check if it can be accessed first)
If it's not valid we show an error message below the input.

Maybe less elegant: we have a "use"-button after the url to confirm.

Reply:
(for serlo I'd honestly would like to restrict this to only images coming from serlo.org in the long term)

Reply:
> There should be a validation of the entered url. This also applies to copy pasting.
> Validation should at least check if the url is structured like a valid image link.
> (starting with "http" and ending with a supported file format like "jpg" for example)
> (but maybe also only allow images from selected domains or fetch the image to check if it can be accessed first)
> If it's not valid we show an error message below the input.

I like that!
I will design an error message tomorrow!

Reply:
Should I make a new Issue or just leave it here?

Reply:
<img width="792" alt="Screenshot 2024-06-26 at 19 34 39" src="https://github.com/serlo/frontend/assets/122027895/5740cf47-c99a-4669-b079-e1f72c20562d">
<img width="792" alt="Screenshot 2024-06-26 at 19 34 53" src="https://github.com/serlo/frontend/assets/122027895/9c0c8940-a010-46c1-943d-16d2367eec50">

Error Message German:
Fehler: Ungültige oder unvollständige URL

Tooltip:
Die eingegebene URL ist entweder ungültig oder unvollständig. Bitte stelle sicher, dass du die vollständige URL korrekt kopiert und eingefügt hast. Die URL sollte mit "http://" oder "https://" beginnen.

Error Message English:
Error: Invalid or Incomplete URL

Tooltip:
The URL you entered is either invalid or incomplete. Please ensure you have copied and pasted the full URL correctly. The URL should start with "http://" or "https://".

Font: Karla
Font Size: 14
Weight: Bold (Error Message), Medium (Tooltip)

Color (For Border and Error Message): Use Serlo Color Scheme 
(If not available: #FF0000, 70% Opacity)

Reply:
Done